src/main/kotlin/no/nav/familie/ef/sak/service/VurderingStegService.kt

```diff
@@ -95,9 +104,27 @@ class VurderingStegService(private val behandlingService: BehandlingService,
             stegService.håndterVilkår(behandling).id
         } else if (behandling.steg != StegType.VILKÅR && vilkårsresultat.any { it == Vilkårsresultat.IKKE_TATT_STILLING_TIL }) {
             stegService.resetSteg(behandling.id, StegType.VILKÅR)
+        } else if (erInitiellVurderingAvVilkår(behandling)) {
```

Comment:
Dette virket som en smidig måte for å håndtere start behandling.
Idag setter `VilkårSteg` behandling til `UTREDES`, burde det slettes/endres?

Reply:
Tja, kanskje. Lite som skjer om det blir satt en ekstra gang, men det er vel unødvendig?

Reply:
Ja, tenkte bare att det kan være litt uklart en annen dag hvorfor den settes 2 ganger :P Men er ikke en big deal